docs/store-operations/catalog/graphql-admin/product-basic-info.mdx

```diff
@@ -33,6 +33,7 @@ The following table describes the ID fields that you can use in inputs for mutat
 | | Shared modifier ID | Navigate to **Products <br /> >Product Options <br /> >Shared Modifiers <br /> >_ModifierName_** <br /> in the control panel. <br /><br /> Retrieve the global ID from the URL. | `"bc/store/sharedProductModifier/2"` |
 | `valueId` | Variant option value ID | [Get all product variant option values](/docs/rest-catalog/product-variant-options/values#get-all-product-variant-option-values) | Product variant options: `"bc/store/productOptionValue/68"` <br /><br />Shared variant options: `"bc/store/sharedProductOptionValue/123"` |
 | | Modifier value ID | [Get all modifier values](/docs/rest-catalog/product-modifiers/values#get-all-modifier-values) | Product modifiers: `"bc/store/productModifierValue/107"` <br /><br />Shared modifiers: `"bc/store/sharedProductModifierValue/107"` |
+| `customFieldId` | ID for a product custom field | [Get product custom fields](/docs/rest-catalog/products/custom-fields#get-product-custom-fields) | `"bc/store/productCustomField/1"` |
```

Comment:
@bc-andreadao it might be worth to include links to the API to query the data in a context of a channel and locale here as well, for the events when webhook payload has a context in it. We can address this latter as a separate ticket and add this for product, options, modifiers webhooks.